Sources/Root/FilterRootViewController.swift

```diff
@@ -418,24 +426,28 @@ extension FilterRootViewController: FilterCellDelegate {
     }
 }
 
-extension FilterRootViewController: SearchQueryCellDelegate {
-    func searchQueryCellDidTapSearchBar(_ searchQueryCell: SearchQueryCell) {
-        guard let indexPath = tableView.indexPath(for: searchQueryCell) else {
-            return
-        }
-        guard let searchQueryFilterInfo = self.filterInfo(at: indexPath) as? SearchQueryFilterInfoType else {
-            return
-        }
-        navigator.navigate(to: .searchQueryFilter(filterInfo: searchQueryFilterInfo, delegate: self))
+extension FilterRootViewController: SearchViewControllerDelegate {
+    public func presentSearchViewController(_ searchViewController: SearchQueryViewController) {
+        searchViewController.searchQuerySuggestionDataSource = searchQuerySuggestionDataSource
+        view.addSubview(searchViewController.view)
+        searchViewController.view.fillInSuperview()
+        view.layoutIfNeeded()
+        // Expand bottom sheet if needed
+        guard let presentationController = navigationController?.presentationController as? BottomSheetPresentationController else { return }
+        searchViewController.previousSizeMode = presentationController.currentContentSizeMode
+        guard presentationController.currentContentSizeMode == .compact else { return }
```

Comment:
It is much nicer if the else is on a newline, makes it easier to read (according to me at least), but primarily it is much easier to debug the code later since you can put a breakpoint there to see that you are exiting early. 

That ^ goes for the rest of the guard-else one-liners too 😄

Reply:
I'll fix it ;)